src/status_im/events.cljs

```diff
@@ -60,6 +60,7 @@
             [status-im.wallet.core :as wallet]
             [status-im.wallet.db :as wallet.db]
             [status-im.web3.core :as web3]
+            [status-im.chat.status-go.core :as chats-go]
```

Comment:
there should not be a status-go namespace

Reply:
for rpc use json-rpc directly
for direct calls we already have `native-module`

Reply:
yean, naming is bad, but I wanted to collect all the effects related to protocol on status-go side to one namespace. idk how to name it better, `chat-rpc`?

Reply:
I think json-rpc is still the place to go you are going to have something similar to `invoke-api` above which won't bring anything and will call json-rpc anyway, after reworking all the rpc calls in the app I've came to the conclusion that it is better to call json-rpc directly it is barely more verbose (most params in the map are optional anyway) and makes need for optimization and new api-calls more visible for instance

https://github.com/status-im/status-react/blob/develop/src/status_im/ethereum/transactions/core.cljs#L111-L139

Reply:
great, thanks for an example, I'll take a looks and change stuff accordingly